Makefile

```diff
@@ -40,6 +40,6 @@ start:
 build: check-env clone
 	@ printf "\n[${GREEN} Generating environment variables... ${NO_COLOR}]\n"
 	cd ${SDX_HOME}/sdx-common && pip install .; cd -
-	cd ${SDX_HOME}/sdx-ops && ${PYTHON3} sdx/ops/configure.py --env > ${SDX_HOME}/sdx-compose/env/private.env ;cd -
+	cd ${SDX_HOME}/sdx-ops && python sdx/ops/configure.py --env > ${SDX_HOME}/sdx-compose/env/private.env ;cd -
```

Comment:
The bash script is probably the way to go as Warran suggests 👍 

I would just check keys in, but even if they're technically fake and not used for anything there's always a risk one day they _may_ actually get used, plus any security scans we do of our repos would probably be false-positiving by picking them up.